src/hooks/useWithdrawFormState.ts

```diff
@@ -216,8 +216,8 @@ export default function useWithdrawFormState(
       }
       setFormState((prevState) => ({
         ...prevState,
-        error: null,
         ...nextState,
+        error: null,
```

Comment:
We do want to clear out the previous state's error, but we don't want to overwrite the nextState's error - was this a suggested change?